crates/editor/src/actions.rs

```diff
@@ -391,3 +390,5 @@ gpui::actions!(
 action_as!(outline, ToggleOutline as Toggle);
 
 action_as!(go_to_line, ToggleGoToLine as Toggle);
+
+action_as!(editor, OpenSelectedFilename as [OpenFile]);
```

Comment:
I think it's confusing to use `as` for these two very different meanings.  Not sure what to do instead though.  Maybe just a new macro that takes a list of string aliases?

An alternative might also be to clarify that these aliases are for use in configuration files.  We may want to even deprecate them / auto-migrate configs off them to encourage homogeneity.

Reply:
Pushed a fresh attempt, using deprecation metadata in the JSON Schema we provide for keymap files so that the language server will surface this information in a helpful way. I did not try to implement auto-migration.